### PR TITLE
Reworking fstab service ordering

### PR DESCRIPTION
**Pull request by jbaublitz**
I want to test this a little bit before merge.

### CODE REVIEW

**Comment by jbaublitz** [comment]
Supersedes #3437

File: systemd/stratis-fstab-setup@.service.in [hunk 1]
```diff
@@ -3,6 +3,8 @@ Description=Set up Stratis filesystems in /etc/fstab
 Documentation=man:stratisd(8)
 DefaultDependencies=no
 Requires=stratisd-min-postinitrd.service
+After=stratisd-min-postinitrd.service
+Before=pre-local-fs.target
```
Comment (bmr-cymru) [inline systemd/stratis-fstab-setup@.service.in:7]: This should be `local-fs-pre.target`.

File: systemd/stratisd-min-postinitrd.service.in [hunk 2]
```diff
@@ -4,7 +4,7 @@ Documentation=man:stratisd(8)
 DefaultDependencies=no
 After=systemd-udevd.service
 Requires=systemd-udevd.service
-Before=local-fs.target
+Before=pre-local-fs.target
```
Comment (bmr-cymru) [inline systemd/stratisd-min-postinitrd.service.in:7]: Ditto

**Review by bmr-cymru — CHANGES_REQUESTED** [review]

File: systemd/stratis-fstab-setup@.service.in (hunk 1 above)
Comment (jbaublitz) [reply systemd/stratis-fstab-setup@.service.in]: Whoops! Fixed.

File: systemd/stratisd-min-postinitrd.service.in (hunk 2 above)
Comment (jbaublitz) [reply systemd/stratisd-min-postinitrd.service.in]: Fixed.

**Review by bmr-cymru — APPROVED** [review]

**Review by mulkieran — APPROVED** [review]

**Pull request merged**